services/bookstore/cmd/controller/main.go

```diff
@@ -42,6 +43,10 @@ func main() {
 	var ackCfg ackrt.Config
 	ackCfg.BindFlags()
 	flag.Parse()
+	zapOptions := zap.Options{
+		Development: ackCfg.EnableDevModeLogging,
+	}
+	ctrlrt.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
```

Comment:
We will want to hide all of this inside an `ackCfg.SetupLogger()` method. The benefits of doing that are:

* No need to import the zap library above
* There will only be a single call added to the main.go template and all generated controllers (easier maintenance)